controllers/managedfusionoffering_controller.go

```diff
@@ -167,7 +169,7 @@ func (r *ManagedFusionOfferingReconciler) reconcileSubscription() error {
 		if err := r.own(r.subscription); err != nil {
 			return err
 		}
-		desiredSubscriptionSpec := pluginGetDesiredSubscriptionSpec(r)
+		desiredSubscriptionSpec := pluginGetDesiredSubscriptionSpec(r.managedFusionOffering)
```

Comment:
Not sure why you decided to stop sending the reconciler, please add it back
```suggestion
		desiredSubscriptionSpec := pluginGetDesiredSubscriptionSpec(r, r.managedFusionOffering)
```